HACKING

```diff
@@ -54,9 +54,10 @@ for systemd (this example is for Fedora):
         $ git clone https://github.com/systemd/systemd.git
         $ cd systemd
         $ vim src/core/main.c                  # or wherever you'd like to make your changes
-        $ dnf builddep systemd                 # install build dependencies
         $ ./autogen.sh c                       # configure the source tree
         $ make -j `nproc`                      # build it locally, see if everything compiles fine
+        $ sudo dnf builddep systemd            # install build dependencies
+        $ sudo dnf install mkosi
```

Comment:
this change doesn't look right. You should really install the systemd build deps *before* invoking "autogen.sh c", otherwise the deps will be most likely missing